src/store/state/board_instance_add.go

```diff
@@ -81,7 +81,28 @@ func (bi *BoardInstance) NewPost(post *r0.Post) (uint64, error) {
 		// Add post to board page.
 		tpRef, tPage, e := pages.BoardPage.GetThreadPage(tpHash, nil)
 		if e != nil {
-			return e
+			// TODO: Fix bug. Workaround...
+			// <<< START : WORKAROUND >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+
+			var doneError = errors.New("done")
+			e := pages.BoardPage.Threads.Ascend(func(i int, tpRefGot *skyobject.RefsElem) (err error) {
```

Comment:
Has nothing to do with the problem. Can ignore

---

I adivse to use underscore if argument never used. It's better for reading own code later.
```go
e := pages.BoardPage.Threads.Ascend(func(_ int, tpRefGot *skyobject.RefsElem) (err error) {
```
However, this is mostly a matter of taste.

Reply:
Which part to ignore? If I don't have this, I can't find the required element??

Reply:
Ignore my comment, =)

Reply:
Are you using `HashTableIndex` for this `Refs`? `RefByHash` uses index if the flag set and fall-backs to `Descend` if the flag is not set. You are using `Ascend` in the workaround. May be my `Descend` works wrong. May be not.

Reply:
I'll have a look